WooCommerce/Classes/ViewRelated/Dashboard/Analytics Hub/AnalyticsHubView.swift

```diff
@@ -94,6 +96,43 @@ private extension AnalyticsHubView {
     }
 }
 
+/// This view simulates pull-to-refresh support on ScrollView before iOS 16.
+///
+/// On iOS 16+ it uses ScrollView.
+/// On iOS 15 it falls back to List with modifiers removing all its default rows styling.
+///
+private struct RefreshablePlainList<Content: View>: View {
```

Comment:
Do you think we should move this to a public reusable view? Seems quite handy!

Reply:
Moved it to `/SwiftUI Components` in f32377a.